charts/helper-console-links/templates/ConsoleLink.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleLink
+metadata:
+  name: {{ .Values.name }}
+spec:
+  applicationMenu:
+    imageURL: >-
+      {{ .Values.image_url }}
+    section: Tools
```

Comment:
should `tools` be configurable too?

Reply:
`location` and `section` can be configurable I think..in case one would like to have a section like `ThirdParty Applications` or add links under `HelpMenu` location.

Reply:
yup yup - and sensible defaults for this would be a good place to strat. 

@JuozasA - should you add the change you were originally making too to include this helper chart with argocd too or do you want to wait and make subsequent PRs?